activesupport/test/deprecation_test.rb

```diff
@@ -152,11 +152,32 @@ def test_default_silence_behavior
     behavior = ActiveSupport::Deprecation.behavior.first
 
     stderr_output = capture(:stderr) {
-      assert_nil behavior.call("Some error!", ["call stack!"])
+      assert_nil behavior.call("Some error!", ["call stack!"], "horizon", "gem")
     }
     assert stderr_output.empty?
   end
 
+  def test_default_notify_behavior
+    ActiveSupport::Deprecation.behavior = :notify
+    behavior = ActiveSupport::Deprecation.behavior.first
+
+    begin
+      events = []
```

Comment:
Why define this as an array when we expect there to only be one? If it wasn't the right message, we'd eventually find out in the latter tests.

E.g. we could opt for `_, event = ActiveSupport::Notifications.subscribe("deprecation.my_gem.custom", &:itself)`.